index.html

```diff
@@ -2309,6 +2416,22 @@ <h5 id="Engineer Traits">Engineer Traits</h5>
                   <td>Engineer Level 1
                   </td>
                </tr>
+               <tr>
+                  <td>Alarm
+                  </td>
+                  <td>
+                     You may place a tripwire across 15ft.  When these are crossed by a creature, 
+                     you are aware they've been set off and the size of the creature which triggered them.
+                     This wakes you up if during a long rest, and they persist until you deconstruct them,
+                     they are triggered, or till the end of a long rest.
+
+                     These can be bypassed by a creature which is intelligent and perceptive enough.
```

Comment:
Very cool, maybe have an option to set it up so it makes a loud noise as an alternative. Also on the very last bit we should explicitly state the passive perception/check needed to spot it - probably INT value DC.